src/index.ts

```diff
@@ -315,9 +328,31 @@ class Monika extends Command {
                 config?.notifications
               )
             })
+
+            scheduledTasks.push(scheduledTlsCheckTask)
           })
         }
 
+        // schedule status update notification
+        if (
+          process.env.NODE_ENV !== 'test' &&
+          flags['status-notification'] !== 'false'
+        ) {
+          // defaults to 6 AM
```

Comment:
Do we have any (special) reason for set it as 6 AM?

Reply:
6 AM is the default defined in the requirement

<img width="541" alt="Screen Shot 2021-08-25 at 15 17 21" src="https://user-images.githubusercontent.com/22452017/130754183-59487140-5654-480f-8fb5-8c7a6149ab84.png">